### ISSUE TITLE
Configure VADS import

### ISSUE THREAD

**Issue by ryguyk** [department-of-veterans-affairs/va.gov-team#97911]
**Note:** This is likely a Platform task, but capturing the need here for now.

## Description

This relates to

**Comment by Karak888** [comment]
I am going to archive this ticket so that we have for reference, but trying to clean up.  We could also move this to your decision log, but I think you already have that covered @ryguyk

**Comment by humancompanion-usds** [comment]
@ryguyk - Did you end up using any of the Design System tokens in any of our work?

**Comment by ryguyk** [comment]
> [@ryguyk](https://github.com/ryguyk) - Did you end up using any of the Design System tokens in any of our work?

@humancompanion-usds I have been using tokens from here: https://unpkg.com/@department-of-veterans-affairs/css-library@0.16.0/dist/tokens/css/variables.css

This is not the newest version. I know the newest version has added at least spacing variables. Currently, I'm using `unit-*` variables for margins/padding/etc. Whether those should be replaced with `vads-spacing-*` variables is debatable, but it's not on my priority list at this point to make the switch or go back and update. 

In short, we're using tokens, but it's not a perfect science. They have certainly been helpful to some degree, though.
